tests/integration_tests/pretrained_models_test.py

```diff
@@ -220,613 +219,634 @@ def setUp(self) -> None:
         self.assertAlmostEqual(res, self.coco_segmentation_subclass_pretrained_mious["shelfnet34_lw"], delta=0.001)
 
     def test_pretrained_efficientnet_b0_imagenet(self):
-        trainer = Trainer('imagenet_pretrained_efficientnet_b0',
-                          multi_gpu=MultiGPUMode.OFF)
+        trainer = Trainer("imagenet_pretrained_efficientnet_b0", multi_gpu=MultiGPUMode.OFF)
 
-        model = models.get("efficientnet_b0", arch_params=self.imagenet_pretrained_arch_params["efficientnet_b0"],
-                           **self.imagenet_pretrained_ckpt_params)
-        res = trainer.test(model=model, test_loader=self.imagenet_dataset, test_metrics_list=[Accuracy()],
-                           metrics_progress_verbose=True)[0].cpu().item()
+        model = models.get("efficientnet_b0", arch_params=self.imagenet_pretrained_arch_params["efficientnet_b0"], **self.imagenet_pretrained_ckpt_params)
+        res = trainer.test(model=model, test_loader=self.imagenet_dataset, test_metrics_list=[Accuracy()], metrics_progress_verbose=True)[0].cpu().item()
         self.assertAlmostEqual(res, self.imagenet_pretrained_accuracies["efficientnet_b0"], delta=0.001)
 
     def test_transfer_learning_efficientnet_b0_imagenet(self):
-        trainer = Trainer('imagenet_pretrained_efficientnet_b0_transfer_learning',
-                          multi_gpu=MultiGPUMode.OFF)
-
-        model = models.get("efficientnet_b0", arch_params=self.imagenet_pretrained_arch_params["efficientnet_b0"],
-                           **self.imagenet_pretrained_ckpt_params, num_classes=5)
-        trainer.train(model=model, training_params=self.transfer_classification_train_params,
-                      train_loader=self.transfer_classification_dataloader,
-                      valid_loader=self.transfer_classification_dataloader)
+        trainer = Trainer("imagenet_pretrained_efficientnet_b0_transfer_learning", multi_gpu=MultiGPUMode.OFF)
+
+        model = models.get(
+            "efficientnet_b0", arch_params=self.imagenet_pretrained_arch_params["efficientnet_b0"], **self.imagenet_pretrained_ckpt_params, num_classes=5
+        )
+        trainer.train(
+            model=model,
+            training_params=self.transfer_classification_train_params,
+            train_loader=self.transfer_classification_dataloader,
+            valid_loader=self.transfer_classification_dataloader,
+        )
 
     def test_pretrained_ssd_lite_mobilenet_v2_coco(self):
-        trainer = Trainer('coco_ssd_lite_mobilenet_v2',
-                          multi_gpu=MultiGPUMode.OFF)
-        model = models.get("ssd_lite_mobilenet_v2",
-                           arch_params=self.coco_pretrained_arch_params["ssd_lite_mobilenet_v2"],
-                           **self.coco_pretrained_ckpt_params)
+        trainer = Trainer("coco_ssd_lite_mobilenet_v2", multi_gpu=MultiGPUMode.OFF)
```

Comment:
Did you modify the checkpoint in our pretrained models bucket? Does the SSD tests here all pass?

Reply:
Yes, renamed the old one instead. Tests pass. Or to be more specific, v2 does, while v1 didn't pass even before my changes. mAP of v1 is 0 because the pre-trained weights are for a dataset with [0, 1] normalization